model/client4_test.go

```diff
@@ -56,3 +56,14 @@ func TestClient4CreatePost(t *testing.T) {
 	_, resp := client.CreatePost(post)
 	assert.Equal(t, http.StatusOK, resp.StatusCode)
 }
+
+func TestClient4SetToken(t *testing.T) {
+	client := NewAPIv4Client("http://example.com")
+	token := NewId()
+
+	client.AuthToken = token
```

Comment:
Hmm, this doesn't actually test `SetToken`, as it does the work of that function itself. Can we change this to call `SetToken`, and then add a second unit test to verify the continued functionality of `MockSession`?

Reply:
Added a test to make sure MockSession works correctly. I tried to use the TestHelper in `model` package, but instead use httptest to check token because of import cycle.